### DISCUSSION TITLE
sudo ssh

### DISCUSSION THREAD

**Discussion by voronczoff** [elfmz/far2l#1749] [Q&A] [answered]
Is it possible to connect via ssh as sudo bash?

**Comment by unxed** [comment, marked as answer]
Yes, by specifying custom sftp server in connection settings

**Comment by voronczoff** [comment]
thank you, i did not find it in SCP type, but in SFTP there is Custom subsystem exec.
sudo /bin/bash does not work for some reason as in windows far
but this works:
sudo /usr/lib/openssh/sftp-server
